pandas/tests/io/formats/test_info.py

```diff
@@ -82,18 +55,46 @@ def test_info_categorical_column():
     df2.info(buf=buf)
 
 
-def test_info(float_frame, datetime_frame):
+@pytest.mark.parametrize(
+    "fixture_func",
+    [
+        int_frame,
+        float_frame,
+        datetime_frame,
+        duplicate_columns_frame,
+    ],
+)
+def test_info_smoke_test(fixture_func, request):
+    frame = request.getfixturevalue(fixture_func.__name__)
     io = StringIO()
-    float_frame.info(buf=io)
-    datetime_frame.info(buf=io)
+    frame.info(buf=io)
```

Comment:
can you do `result = io.getvalue()` and assert *somthing* about the result (not the whole thing)

Reply:
e.g. assert len(result.splitlines()) > 100 or something

Reply:
Done.